docs/use-the-network/devices/ready-to-use/dragino-lht65.mdx

```diff
@@ -72,8 +72,24 @@ manual for instructions.
 
 If you would like to easily view your device's data on a web or mobile dashboard
 for free, this device is supported out of the box by myDevices Cayenne via the
-integration on Console. You can learn how to quickly setup this integration
-here.
+integration on Console. 
+
+To add the LHT65 to myDevices Cayenne, you do not need to set up a Decode script
+as described below. Cayenne takes the encoded payload directly and parses the
+temperature and other data elements from it.
+
+Be careful to select **Helium** when adding the LHT65 to Cayenne.
+As described [here](https://docs.mydevices.com/docs/lorawan/helium), select
+<ol>
+  <li>"Add new..."</li>
+  <li>"Device/Widget"</li>
+  <li>"LoRa"</li>
+  <li>"Helium"<li>
+  <li>select Dragino LHT65</li>
+</ol>
```

Comment:
```suggestion
1. "Add new..."
2. "Device/Widget"
3. "LoRa"
4. "Helium"
5. select Dragino LHT65
```
Since this is mdx, you can use markdown syntax for an ordered list.